src/util/pg.py

```diff
@@ -1621,3 +1622,83 @@ def create_id_sequence(cr, table, set_as_default=True):
                 table=table_sql,
             )
         )
+
+
+def bulk_update_table(cr, table, columns, mapping, key_col="id"):
+    """
+    Update table based on mapping.
+
+    Each `mapping` entry defines the new values for the specified `columns` for the row(s)
+    whose `key_col` value matches the key.
+
+    .. example::
+
+       .. code-block:: python
+
+          # single column update
+          util.bulk_update_table(
+              cr,
+              "account_move",
+              "always_tax_eligible",
+              {
+                  1: True,
+                  2: False,
+              },
+          )
```

Comment:
The examples take a lot of real space in the docs. We can make it shorter
```suggestion
          util.bulk_update_table(cr, "res_users", "active", {1: False, 2: True})
```